### PR TITLE
feat: /internet-of-things/appstore redesgin

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied redesign as per [doc](https://docs.google.com/document/d/1CKwmPDDtXOIgw-jSZFwzP-qLMc_rS_dUACpHQXlg0fA/edit?tab=t.0) and [design](https://www.figma.com/design/1CdIT6TV3ASf55GsnQcwXg/Internet-of-Things?node-id=0-1&node-type=canvas&m=dev)

## QA

- View the site locally in your web browser at: https://ubuntu-com-14523.demos.haus/internet-of-things/appstore
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-16630

### CODE REVIEW

**Comment by eliman11** [comment]
Thanks @mtruj013! Looks good to me, some changes to change to US spelling - I've marked them on the copydoc too if it's easier to spot that way. Feel free to add UX+1 once these are changed.
- [ ] 'Specialized' under 'Ideal for secure software distribution'
- [ ] 'Minimize' under 'Reliable over-the-air (OTA) updates' (subheading)
- [ ] 'Maximize' under 'Maximize bandwidth and speed by caching downloads and snap traffic.'
- [ ] 'Specializing' under 'IoT Professional Services'

Also some of the sections are missing dividers but will leave that with @mattea-turic :)
Screenshot 2024-12-02 at 17 40 24

**Comment by mattea-turic** [comment]
ty @mtruj013 ! a few comments from me:

for this section:
Screenshot 2024-12-03 at 11 17 11
- [ ] hrs between the list items pls

for this section:
Screenshot 2024-12-03 at 11 18 25
- [ ] muted hr above the equal height row

for this section:
Screenshot 2024-12-03 at 11 20 58
- [ ] dividers between the list items, incl. above the first one
- [ ] spacing is missing between the text, "ready to find out more" and the button

for this section:
Screenshot 2024-12-03 at 11 23 57
- [ ]  hrs missing at the bottom of each container (see figma maybe since i'm not sure i'm describing this well)

for this section:
Screenshot 2024-12-03 at 11 28 12
- [ ] the first quote seems to have more bottom padding than the second one. Could you make it so that they're each wrapped in just a shallow?
On mobile, it might be worth to omit the images as they end up taking too much space (unless this can be swapped with a size more compatible with small screens; I can provide the assets though can you switch between variants like that depending on breakpoint?):
Screenshot 2024-12-03 at 11 35 57

And for this section (mobile):
Screenshot 2024-12-03 at 11 39 38
- [ ]  is it possible to minimise the spacing between the content in the card and the link, since the cards are stacked vertically, so they don't have to occupy the same length (unlike on desktop)

**Comment by mtruj013** [comment]
Thanks @eliman11 and @mattea-turic ! I've addressed most of the comments with the exception of the following:

> hrs missing at the bottom of each container (see figma maybe since i'm not sure i'm describing this well) 

Sorry, I'm not seeing what you mean. The demo seems to match the figma:
demo:
image

figma:
image

Not including hrs for each link is also consistent with how we use this design elsewhere, examples [here](https://canonical.com/data/valkey) and [here](https://canonical.com/data/kafka/support).

> On mobile, it might be worth to omit the images as they end up taking too much space (unless this can be swapped with a size more compatible with small screens; I can provide the assets though can you switch between variants like that depending on breakpoint?

Yes please, the assets currently have a lot of white space baked in so reducing the size makes the text illegible on smaller screens:
image

Should have no/very minimal whitespace to format correctly, eg from u.com/aws:
image

**Comment by mattea-turic** [comment]
ty @mtruj013 ! Here are the assets for [screenly](https://assets.ubuntu.com/v1/3cd4128e-screenly.png), [domotz](https://assets.ubuntu.com/v1/3c4beadc-domotz.png), and [rigado](https://assets.ubuntu.com/v1/e512541f-rigado.png)

**Comment by eliman11** [comment]
Looks good, thank you! Adding +1

**Comment by mattea-turic** [comment]
Looks great, thanks @mtruj013 ! I'll add a +1;
there's one v small detail that'll need to be changed, but I think this is a super minor fix, where there's an extra hr above "onboard new devices" on smaller screens
Screenshot 2024-12-04 at 09 50 07

**Comment by immortalcodes** [comment]
image
image

Design is a bit different here, but I guess it is already talked out?

**Comment by mattea-turic** [comment]
> Design is a bit different here, but I guess it is already talked out?

Ah I don't remember that changing on the figma; the demo is how I had initially intended it, so all good there

**Comment by immortalcodes** [comment]
image
Double lines on small screen

File: templates/internet-of-things/appstore.html [hunk 1]
```diff
@@ -112,238 +132,355 @@ <h2>Check out the dedicated Snap Store</h2>
...
+          <p>Access the case study&nbsp;&rsaquo;</p>
+        </a>
+      </div>
+      <div class="col-3 col-medium-2 col-small-2">
+        <a href="/dell" aria-label="Dell Technologies">
+          <div class="p-image-container is-highlighted">
+            {{ image(url="https://assets.ubuntu.com/v1/54e5c806-dell-technologies.png",
+                        alt="",
+                        width="852",
+                        height="481",
```
[the remark this answers is not in the document]
Comment (mtruj013) [reply templates/internet-of-things/appstore.html:227]: Nah, that's what the aria-label on the link that's wrapping the image is for 🙂 Original ux/a11y feedback for this pattern [here](https://github.com/canonical/canonical.com/pull/1386#issuecomment-2397136348)

**Pull request merged**